tools/packages.config

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-	<package id="Cake" version="0.10.1" />
+	<package id="Cake" version="0.17.0" />
```

Comment:
I had to update cake to add wix support. If you have a build server you will need to clean your packages directory so you download a new version of cake